checkov/terraform/checks/graph_checks/azure/AzureSqlDbEnableTransparentDataEncryption.yaml

```
@@ -0,0 +1,19 @@
+metadata: 
+  id: "CKV2_AZURE_25"
+  name: "Ensure Azure SQL database Transparent Data Encryption (TDE) is enabled"
+  category: "GENERAL_SECURITY"
+
+definition:
+  or:
+
+  - cond_type: attribute
+    resource_types: "azurerm_mssql_database"
+    attribute: "transparent_data_encryption_enabled"
+    operator: "not_exists"
+
+  - cond_type: attribute
+    resource_types: "azurerm_mssql_database"
+    attribute: "transparent_data_encryption_enabled"
+    operator: "equals"
+    value: "true"
```

```suggestion
  cond_type: attribute
  resource_types: "azurerm_mssql_database"
  attribute: "transparent_data_encryption_enabled"
  operator: "not_equals"
  value: "false"
```
I'm not 100% sure, but shouldn't be this enough?

---

Yep this will do. I got confused with another policy 😅